clang/include/clang/Driver/Action.h

```diff
@@ -705,6 +714,47 @@ class BackendCompileJobAction : public JobAction {
   }
 };
 
+// Represents a file table transformation action. The order of inputs to a
+// FileTableTformJobAction was constructed with must accord with the tforms
+// added later - some tforms "consume" inputs. For example, "replace column"
```

Comment:
I'm not parsing this comment - can you re-word?  `was constructed with` is not reading well.

Reply:
ok